### PR TITLE
added DB week3 homework

### PR DESCRIPTION
**Pull request by krithi-kondana**

### CODE REVIEW

File: databases/week3/mongo_db/mongo_db.txt [hunk 1]
```diff
@@ -0,0 +1,377 @@
+> use Todo
+switched to db Todo
+
+
+> db.task.insert([{"title":"Wash clothes", "description":"Title says it all.", "created":"2017-10-25 06:54:16", "updated":"2017-10-15 13:05:09", "due_date":""},)
```
Comment (babakbandpey) [inline databases/week3/mongo_db/mongo_db.txt]: The ending prantese breaks the code

File: databases/week3/part2_mealapp/part2_meal_app_queries.sql [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+-- 1.Get meals that has a price smaller than a specific price fx 90
+select *
+from meal
+where price < '70';
+
+-- 2.Get meals that still has available reservations
+select meal.title, meal.description, meal.max_reservations, reservation.number_of_guests, meal.max_reservations-reservation.number_of_guests as available_reservations
+from meal inner join reservation on reservation.meal_id = meal.id
+where meal.max_reservations > reservation.number_of_guests;
+
```
Comment (babakbandpey) [inline databases/week3/part2_mealapp/part2_meal_app_queries.sql]: This one does not produce the answer
Please read my comment on the channel. Thank you

File: databases/week3/mongo_db/mongo_db.txt (hunk 1 above)
Comment (krithi-kondana) [reply databases/week3/mongo_db/mongo_db.txt]: corrected

File: databases/week3/part2_mealapp/part2_meal_app_queries.sql (hunk 2 above)
Comment (babakbandpey) [reply databases/week3/part2_mealapp/part2_meal_app_queries.sql]: Do you need help with this?

Comment (krithi-kondana) [reply databases/week3/part2_mealapp/part2_meal_app_queries.sql]: implemented feedback

**Pull request merged**